sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/EventProcessor{TPartition}.cs

```diff
@@ -304,7 +308,9 @@ internal EventProcessorStatus Status
             RetryPolicy = options.RetryOptions.ToRetryPolicy();
             Options = options;
             EventBatchMaximumCount = eventBatchMaximumCount;
+#pragma warning disable CA2214 // Do not call overridable methods in constructors.  The virtual methods are internal and used for testing.
```

Comment:
nit:  Do you mind adding a leading space to this, just to give some visual separation before the pragma block?